src/components/Auth/Modal/AuthModalLogin/AuthModal.tsx

```diff
@@ -38,6 +43,11 @@ function AuthModal({ onCloseModal }: Readonly<LoginModalProps>) {
     )
   }
 
+  const handleForgotPasswordClick = () => {
```

Comment:
I think the function handleForgotPasswordClick is unnecessary.
the resetOpenModal() function can be called in the onClick event of the Link component.